diff -Nru LPRng-3.7.2/configure.in LPRng-3.7.2.new/configure.in
--- LPRng-3.7.2/configure.in	Thu Dec 21 20:42:25 2000
+++ LPRng-3.7.2.new/configure.in	Thu Dec 21 20:45:20 2000
@@ -13,46 +13,13 @@
 AC_CANONICAL_SYSTEM
 AC_CONFIG_HEADER(config.h)
 
-AC_DEFINE_UNQUOTED(PACKAGE, "$PACKAGE")
-AC_DEFINE_UNQUOTED(VERSION, "$VERSION")
-AC_SUBST(PACKAGE)
-AC_SUBST(VERSION)
-ALL_LINGUAS="fr"
-
-AC_PREFIX_DEFAULT(/usr/local)
-AC_SUBST(INSTALL)
-AC_SUBST(INSTALL_MAN)
-AC_SUBST(SHELL)
-AC_SUBST(CPP)
-AC_SUBST(CFLAGS)
-AC_SUBST(CPPFLAGS)
-AC_SUBST(LDFLAGS)
-AC_SUBST(LDCC)
+AM_INIT_AUTOMAKE(LPRng, 3.7.2)
 
 dnl check for awk
 AC_PROG_AWK
-AC_SUBST(AWK)
-AC_PATH_PROG(PERL,perl)dnl
-AC_SUBST(PERL)dnl
-AC_PATH_PROG(CHOWN,chown)dnl
-AC_SUBST(CHOWN)dnl
-AC_PATH_PROG(CHGRP,chgrp)dnl
-AC_SUBST(CHGRP)dnl
-
-
-WITH_CC dnl
-WITH_CCOPTS dnl
-WITH_LINKER dnl
-WITH_LDOPTS dnl
-WITH_CPPOPTS dnl
-
-if test -z "$LOCALEDIR" ; then
-	LOCALEDIR='$(prefix)/share/locale'
-fi;
-
-WITH_LOCALEDIR dnl
-
-
+AC_PATH_PROG(PERL,perl)
+AC_PATH_PROG(CHOWN,chown)
+AC_PATH_PROG(CHGRP,chgrp)
 
 dnl ----------------------------------------------------------------------------
 dnl ----------- XXXXXX START OF OPTIONS ----------------------------------------
@@ -331,38 +298,11 @@
 AC_MSG_RESULT($FILTER_DIR)
 AC_SUBST(FILTER_DIR)
 
-dnl default strip binaries 
-AC_MSG_CHECKING(disable stripped binaries)
-AC_ARG_ENABLE( strip,
-[  --disable-strip                   disable stripping binaries by default],
-[v=no; STRIP=""; ],
-[v=yes;  STRIP="-s"; ]
-)
-AC_MSG_RESULT($v)
-AC_SUBST(STRIP)
-
-
 dnl ----------------------------------------------------------------------------
 dnl -----------   C compiler checks    -----------------------------------------
 AC_PROG_CC
 AC_PROG_CPP
 
-if test "$ac_cv_prog_gcc" = yes; then
-	cat >conftest.c <<EOF
-#if __GNUC__ < 2 || (__GNUC__ == 2 && __GNUC_MINOR__ < 5)
-yes
-#endif
-EOF
-    if ${CC-cc} -E conftest.c | egrep yes >/dev/null 2>&1; then
-        gcc -v
-		echo "WARNING: gcc -O (version 2.4.5 and below) produces incorrect code"
-		echo "  for lpd.c and other large routines. Update your GCC compiler"
-		exit 1
-	fi;
-fi;
-
-LDCC="$CC"
-
 dnl ----------------------------------------------------------------------------
 dnl ------------   OS Checks  --------------------------------------------------
 AC_ISC_POSIX
@@ -1322,38 +1262,9 @@
 SHELL=/bin/sh
 AC_MSG_RESULT(using $SHELL (FORCED))
 
+ALL_LINGUAS="fr"
 AM_GNU_GETTEXT
 
- if test "${USE_NLS}" = yes; then
-   AC_MSG_CHECKING([Using NLS, localedir ${LOCALEDIR}, ])
-   if test "${USE_INCLUDED_LIBINTL}" = yes ; then
-	  AC_MSG_RESULT([using the local libintl installation])
-	  if test -f ${srcdir}/${nls_cv_header_libgt} ; then
-		INTLINCS='-I $(top_builddir)/intl'
-	  else
-		AC_MSG_RESULT([NLS enabled, local libintl support is needed, and])
-		AC_MSG_RESULT([ source code is not in the ${srcdir}/intl directory.])
-		AC_MSG_RESULT([ See ${srcdir}/ABOUT-NLS.LPRng in the LPRng distribution for details.])
-		exit 1;
-	  fi
-   else
-	  AC_MSG_RESULT([using -lintl])
-	  INTLLIBS=-lintl
-   fi
- else
-   AC_MSG_RESULT([No NLS])
- fi   
- if test "${USE_NLS}" != yes -o "${USE_INCLUDED_LIBINTL}" != yes ; then
-	nls_cv_header_intl=
-	nls_cv_header_libgt=
- fi
- if test "$XGETTEXT" = "" ; then
-	 XGETTEXT=xgettext;
- fi
- AC_MSG_RESULT(using XGETTEXT $XGETTEXT)
- AC_SUBST(INTLINCS)
- AC_SUBST(INTLLIBS)
-
 dnl ----------------------------------------------------------------------------
 dnl ----------------- END OF GENERAL CONFIGURATION   ---------------------------
 
@@ -1489,7 +1400,6 @@
 
 
 AC_OUTPUT(
-[
 Makefile
 HOWTO/Makefile
 TESTSUPPORT/Makefile
@@ -1519,28 +1429,22 @@
 src/psbanner
 po/Makefile.in
 intl/Makefile
-],
-[
-for i in  \
-UTILS/LPRng.pm \
-UTILS/accounting.pl \
-UTILS/decode_args_with_perl \
-UTILS/decode_args_with_sh \
-UTILS/fixid \
-UTILS/fixupdate \
-UTILS/lpq_in_perl \
-UTILS/lpr_in_perl \
-UTILS/lprm_in_perl \
-UTILS/make_lpd_conf \
-UTILS/make_printcap_use \
-UTILS/makeinc \
-UTILS/read_conf \
-UTILS/remote_active \
-UTILS/test_read \
-UTILS/update_z.pl \
-; do chmod +x $i ; done
-]
-
+UTILS/LPRng.pm
+UTILS/accounting.pl
+UTILS/decode_args_with_perl
+UTILS/decode_args_with_sh
+UTILS/fixid
+UTILS/fixupdate
+UTILS/lpq_in_perl
+UTILS/lpr_in_perl
+UTILS/lprm_in_perl
+UTILS/make_lpd_conf
+UTILS/make_printcap_use
+UTILS/makeinc
+UTILS/read_conf
+UTILS/remote_active
+UTILS/test_read
+UTILS/update_z.pl
 )
 
 dnl ----------------------------------------------------------------------------
diff -Nru LPRng-3.7.2/man/Makefile.in LPRng-3.7.2.new/man/Makefile.in
--- LPRng-3.7.2/man/Makefile.in	Thu Dec 21 20:42:25 2000
+++ LPRng-3.7.2.new/man/Makefile.in	Thu Dec 21 20:41:40 2000
@@ -22,8 +22,8 @@
 
 ## fix up prefix to be a make variable
 prefix = @prefix@
-MAN =		@mandir@
-SHELL =		@SHELL@
+mandir = @mandir@
+SHELL =	@SHELL@
 
 # Install man pages in compressed form.
 # COMPRESS     = compression utility
@@ -61,13 +61,13 @@
 install:
 	for suffix in 1 2 3 4 5 6 7 8 ; do \
 	  if ls ${SRC}/*.$${suffix} >/dev/null 2>&1 ; then \
-	    if [ ! -d $(DESTDIR)$(MAN)/man$${suffix} ] ; then \
-		    echo ${SRC}/mkinstalldirs $(DESTDIR)$(MAN)/man$${suffix}; \
-		    ${SRC}/mkinstalldirs $(DESTDIR)$(MAN)/man$${suffix}; \
+	    if [ ! -d $(DESTDIR)$(mandir)/man$${suffix} ] ; then \
+		    echo ${SRC}/mkinstalldirs $(DESTDIR)$(mandir)/man$${suffix}; \
+		    ${SRC}/mkinstalldirs $(DESTDIR)$(mandir)/man$${suffix}; \
 	    fi; \
 	    for i in ${SRC}/*.$$suffix; do \
-		echo   $(INSTALL) -m 644 $$i $(DESTDIR)$(MAN)/man$$suffix/`basename $$i`$(MANEXT); \
-		$(INSTALL)   -m 644 $$i $(DESTDIR)$(MAN)/man$$suffix/`basename $$i`$(MANEXT); \
+		echo   $(INSTALL) -m 644 $$i $(DESTDIR)$(mandir)/man$$suffix/`basename $$i`$(mandirEXT); \
+		$(INSTALL)   -m 644 $$i $(DESTDIR)$(mandir)/man$$suffix/`basename $$i`$(mandirEXT); \
 	    done; \
 	  fi; \
 	done;
@@ -76,8 +76,8 @@
 	for suffix in 1 2 3 4 5 6 7 8 ; do \
 	  if ls ${SRC}/*.$${suffix} >/dev/null 2>&1 ; then \
 	    for i in ${SRC}/*.$$suffix; do \
-		echo   rm $(DESTDIR)$(MAN)/man$$suffix/`basename $$i`$(MANEXT); \
-		rm   $(DESTDIR)$(MAN)/man$$suffix/`basename $$i`$(MANEXT); \
+		echo   rm $(DESTDIR)$(mandir)/man$$suffix/`basename $$i`$(mandirEXT); \
+		rm   $(DESTDIR)$(mandir)/man$$suffix/`basename $$i`$(MANEXT); \
 	    done; \
 	  fi; \
 	done;
diff -Nru LPRng-3.7.2/src/Makefile.in LPRng-3.7.2.new/src/Makefile.in
--- LPRng-3.7.2/src/Makefile.in	Thu Dec 21 20:42:25 2000
+++ LPRng-3.7.2.new/src/Makefile.in	Thu Dec 21 20:41:40 2000
@@ -33,7 +33,8 @@
 @SET_MAKE@
 
 # what C compiler to use.
-CC:=@CC@
+CC=@CC@
+datadir=@datadir@
 
 # what command to use to "ranlib" or "lorder" the library.
 RANLIB=@RANLIB@
@@ -47,15 +48,9 @@
 SHELL_PATH=@SHELL@
 SHELL = @SHELL@
 
-# loader for c programs
-LDCC=@LDCC@
-
 # any libs that need to be used to get these programs to compile:
 LIBS=@INTLLIBS@ @LIBS@
 
-# add path to internationalization
-INTLINCS=@INTLINCS@
-
 ### PATHS ####
 # prefix
 prefix=@prefix@
@@ -72,10 +67,8 @@
 
 # install options
 
-STRIP=@STRIP@
-
 CFLAGS:= @CPPFLAGS@ @CFLAGS@ @DEFS@ \
- -I.. -I${SRC}/include -I${SRC}/common @INTLINCS@ \
+ -I.. -I${SRC}/include -I${SRC}/common \
  -DCHOWN=\"@CHOWN@\"  \
  -DCHGRP=\"@CHGRP@\"  \
  -DPRINTCAP_PATH=\"@PRINTCAP_PATH@\"  \
@@ -88,17 +81,14 @@
  -DLOCKFILE=\"@LOCKFILE@\"  \
  -DCLEAR=\"@CLEAR@\"  \
  -DUSERID=\"@USERID@\"  \
- -DGROUPID=\"@GROUPID@\"  \
- -DLOCALEDIR=\"@LOCALEDIR@\"
-
+ -DGROUPID=\"@GROUPID@\" \
+ -DLOCALEDIR=\"$(datadir)/locale\"
 
 USER_OBJS := @USER_OBJS@
 LDFLAGS   := @LDFLAGS@
 FILTER_DIR = @FILTER_DIR@
 LPD_DIR    = @LPD_DIR@
 
-STRIP      = @STRIP@
-
 #********************** End of configure modifiable settings ************
 
 
@@ -175,7 +165,7 @@
 
 
 SRCDIRS=${SRC}/common ${SRC}/AUTHENTICATE
-INCLUDE=.. ${SRC}/include ${INTLINCS}
+INCLUDE=.. ${SRC}/include
 
 VPATH=..:.:${SRC}/include:${SRC}/common
 
@@ -241,33 +231,33 @@
 	$(RANLIB) $@
 
 lpd: $(LPD_OBJS) liblpr.a
-	$(LDCC) $(LDFLAGS) -o $@ ${LPD_OBJS} liblpr.a $(LIBS)
+	$(CC) $(LDFLAGS) -o $@ ${LPD_OBJS} liblpr.a $(LIBS)
 lpr: $(LPR_OBJS) liblpr.a
-	$(LDCC) $(LDFLAGS) -o $@ ${LPR_OBJS} liblpr.a $(LIBS)
+	$(CC) $(LDFLAGS) -o $@ ${LPR_OBJS} liblpr.a $(LIBS)
 lpq: $(LPQ_OBJS) liblpr.a
-	$(LDCC) $(LDFLAGS) -o $@ ${LPQ_OBJS} liblpr.a $(LIBS)
+	$(CC) $(LDFLAGS) -o $@ ${LPQ_OBJS} liblpr.a $(LIBS)
 lpstat: $(LPSTAT_OBJS) liblpr.a
-	$(LDCC) $(LDFLAGS) -o $@ ${LPSTAT_OBJS} liblpr.a $(LIBS)
+	$(CC) $(LDFLAGS) -o $@ ${LPSTAT_OBJS} liblpr.a $(LIBS)
 lprm: $(LPRM_OBJS) liblpr.a
-	$(LDCC) $(LDFLAGS) -o $@ ${LPRM_OBJS} liblpr.a $(LIBS)
+	$(CC) $(LDFLAGS) -o $@ ${LPRM_OBJS} liblpr.a $(LIBS)
 lpc: $(LPC_OBJS) liblpr.a
-	$(LDCC) $(LDFLAGS) -o $@ ${LPC_OBJS} liblpr.a $(LIBS)
+	$(CC) $(LDFLAGS) -o $@ ${LPC_OBJS} liblpr.a $(LIBS)
 checkpc: $(CHECKPC_OBJS) liblpr.a
-	$(LDCC) $(LDFLAGS) -o $@ ${CHECKPC_OBJS} liblpr.a $(LIBS)
+	$(CC) $(LDFLAGS) -o $@ ${CHECKPC_OBJS} liblpr.a $(LIBS)
 lpf: $(LPF_OBJS)
-	$(LDCC) $(LDFLAGS) -o $@ ${LPF_OBJS} $(LIBS)
+	$(CC) $(LDFLAGS) -o $@ ${LPF_OBJS} $(LIBS)
 lpbanner: $(LPBANNER_OBJS)
-	$(LDCC) $(LDFLAGS) -o $@ ${LPBANNER_OBJS} $(LIBS)
+	$(CC) $(LDFLAGS) -o $@ ${LPBANNER_OBJS} $(LIBS)
 monitor: $(MONITOR_OBJS) liblpr.a
-	$(LDCC) $(LDFLAGS) -o $@ ${MONITOR_OBJS} liblpr.a $(LIBS)
+	$(CC) $(LDFLAGS) -o $@ ${MONITOR_OBJS} liblpr.a $(LIBS)
 
 authenticate_pgp: ${SRC}/AUTHENTICATE/authenticate_pgp.sh
 	rm -f $@; cp ${SRC}/AUTHENTICATE/authenticate_pgp.sh $@; chmod 555 $@
 
 sserver: sserver.o liblpr.a
-	$(LDCC) $(LDFLAGS) -o $@ sserver.o liblpr.a $(LIBS)
+	$(CC) $(LDFLAGS) -o $@ sserver.o liblpr.a $(LIBS)
 sclient: sclient.o liblpr.a
-	$(LDCC) $(LDFLAGS) -o $@ sclient.o liblpr.a $(LIBS)
+	$(CC) $(LDFLAGS) -o $@ sclient.o liblpr.a $(LIBS)
 
 sendauth.o lpd_secure.o :	user_auth.stub
 
@@ -304,17 +294,17 @@
 	  if [ ! -d $$i ] ; then ${SRC}/mkinstalldirs $$i ; fi; \
 	done;
 
-	${INSTALLCMD} ${STRIP} -m $(@PERMS@) lpq ${DESTDIR}${bindir}
-	${INSTALLCMD} ${STRIP} -m $(@PERMS@) lprm ${DESTDIR}${bindir}
-	${INSTALLCMD} ${STRIP} -m $(@PERMS@) lpr ${DESTDIR}${bindir}
-	${INSTALLCMD} ${STRIP} -m $(@PERMS@) lpstat ${DESTDIR}${bindir}
-	${INSTALLCMD} ${STRIP} -m $(@PERMS@) lpc ${DESTDIR}${sbindir}
-	${INSTALLCMD} ${STRIP} -m $(NORM_PERMS) lpd ${DESTDIR}${LPD_DIR}
-	${INSTALLCMD} ${STRIP} -m $(NORM_PERMS) lpf ${DESTDIR}${FILTER_DIR}
-	${INSTALLCMD} ${STRIP} -m $(NORM_PERMS) lpbanner ${DESTDIR}${FILTER_DIR}
-	${INSTALLCMD}    -m $(NORM_PERMS) pclbanner ${DESTDIR}${FILTER_DIR}
-	${INSTALLCMD}    -m $(NORM_PERMS) psbanner ${DESTDIR}${FILTER_DIR}
-	${INSTALLCMD} ${STRIP} -m $(NORM_PERMS) checkpc ${DESTDIR}${sbindir}
+	${INSTALLCMD} -m $(@PERMS@) lpq ${DESTDIR}${bindir}
+	${INSTALLCMD} -m $(@PERMS@) lprm ${DESTDIR}${bindir}
+	${INSTALLCMD} -m $(@PERMS@) lpr ${DESTDIR}${bindir}
+	${INSTALLCMD} -m $(@PERMS@) lpstat ${DESTDIR}${bindir}
+	${INSTALLCMD} -m $(@PERMS@) lpc ${DESTDIR}${sbindir}
+	${INSTALLCMD} -m $(NORM_PERMS) lpd ${DESTDIR}${LPD_DIR}
+	${INSTALLCMD} -m $(NORM_PERMS) lpf ${DESTDIR}${FILTER_DIR}
+	${INSTALLCMD} -m $(NORM_PERMS) lpbanner ${DESTDIR}${FILTER_DIR}
+	${INSTALLCMD} -m $(NORM_PERMS) pclbanner ${DESTDIR}${FILTER_DIR}
+	${INSTALLCMD} -m $(NORM_PERMS) psbanner ${DESTDIR}${FILTER_DIR}
+	${INSTALLCMD} -m $(NORM_PERMS) checkpc ${DESTDIR}${sbindir}
 	cd ${DESTDIR}${bindir}; rm -f lp cancel; ln -s lpr lp; ln -s lprm cancel;
 
 uninstall:
