--- LPRng-3.8.32/Makefile.in~	2007-10-31 14:35:16.000000000 +0100
+++ LPRng-3.8.32/Makefile.in	2007-10-31 14:39:28.000000000 +0100
@@ -132,13 +132,13 @@
 	-rm -f *.bak ? ?.* core *.old *~ po/*~ lprng.sh
 
 install: all
-	if [ ! -d ${DATADIR} ] ; then ${SRC}/mkinstalldirs ${DATADIR}; fi
+	${SRC}/mkinstalldirs $(DESTDIR)${DATADIR}
 	for i in lpd.perms lpd.conf printcap lprng.sh postinstall ; do \
-		${INSTALL} $$i ${DATADIR}/$$i.sample; \
+		${INSTALL} $$i $(DESTDIR)${DATADIR}/$$i.sample; \
 	done
 	OSNAME=@OSNAME@; case "@OSNAME@" in *linux* ) \
 		for i in init.linux init.linuxsb init.redhat ; do \
-		${INSTALL} $$i ${DATADIR}/$$i.sample; \
+		${INSTALL} $$i $(DESTDIR)${DATADIR}/$$i.sample; \
 		done; ;; esac
 	$(MAKE) MAKETARGET=$@ DESTDIR=$(DESTDIR) $(INSTALLDIRS)
 	MAKEINSTALL=YES DESTDIR=$(DESTDIR) $(SHELL) postinstall
