--- lprng-3.9.0/configure.ac.orig	2024-09-20 19:35:58.371780348 +0200
+++ lprng-3.9.0/configure.ac	2024-09-20 19:36:00.945099740 +0200
@@ -948,7 +948,7 @@ if test "$SSL_ENABLE" != ""; then
 	AC_CHECK_LIB([crypto], [RC4_set_key], [true], [SSL_ENABLE=""])
 fi
 if test "$SSL_ENABLE" != ""; then
-	AC_CHECK_LIB([ssl], [SSL_load_error_strings], [true], [SSL_ENABLE=""], [-lcrypto])
+	AC_CHECK_LIB([ssl], [OPENSSL_init_ssl], [true], [SSL_ENABLE=""], [-lcrypto])
 fi
 if test "$SSL_ENABLE" != ""; then
 	SSL_LDADD="-lssl -lcrypto"
