--- LPRng-3.8.15/src/include/portable.h.orig	Mon Aug 12 02:01:47 2002
+++ LPRng-3.8.15/src/include/portable.h	Fri Dec 27 15:43:15 2002
@@ -708,6 +708,7 @@
 #else
 # define _(Text) Text
 # define N_(Text) Text
+# define ngettext(Text, Plural, n) ((n==1)?(Text):(Plural))
 # define textdomain(Domain)
 # define bindtextdomain(Package, Directory)
 #endif
--- LPRng-3.8.15/src/common/lpd_status.c.orig	Mon Aug 12 02:01:45 2002
+++ LPRng-3.8.15/src/common/lpd_status.c	Fri Dec 27 15:47:50 2002
@@ -776,8 +776,8 @@
 	/* this gives a short 1 line format with minimum info */
 	if( displayformat == REQ_DSHORT ){
 		len = safestrlen( header );
-		SNPRINTF( header+len, sizeof(header)-len) _(" %d job%s"),
-			matches, (matches == 1)?"":"s" );
+		SNPRINTF( header+len, sizeof(header)-len) ngettext(" %d job", " %d jobs", matches),
+			matches);
 		if( total_held ){
 			len = safestrlen( header );
 			SNPRINTF( header+len, sizeof(header)-len) _(" (%d held)"), 
@@ -992,8 +992,8 @@
 		safestrncpy( msg, _(" Queue: no printable jobs in queue\n") );
 	} else {
 		/* check to see if there are files and no spooler */
-		SNPRINTF( msg, sizeof(msg)) _(" Queue: %d printable job%s\n"),
-			printable, printable > 1 ? "s" : "" );
+		SNPRINTF( msg, sizeof(msg)) ngettext(" Queue: %d printable job\n",
+			" Queue: %d printable jobs\n", printable), printable );
 	}
 	if( Write_fd_str( *sock, msg ) < 0 ) cleanup(0);
 	if( held ){
