diff -Nru LPRng-3.6.12/configure.in LPRng-3.6.12.new/configure.in
--- LPRng-3.6.12/configure.in	Thu Oct 28 03:30:07 1999
+++ LPRng-3.6.12.new/configure.in	Thu Nov 25 07:41:16 1999
@@ -1255,7 +1255,6 @@
 
 dnl ----------------- Option for enabling GNU gettext --------------------
 AM_GNU_GETTEXT
-test "$USE_NLS" = yes && LIBS="$LIBS -lintl"
 
 dnl ----------------- GCC Compilation ----------------------------------
 AC_OUTPUT(
diff -Nru LPRng-3.6.12/man/Makefile.in LPRng-3.6.12.new/man/Makefile.in
--- LPRng-3.6.12/man/Makefile.in	Thu Nov 25 07:47:59 1999
+++ LPRng-3.6.12.new/man/Makefile.in	Thu Nov 25 07:41:16 1999
@@ -21,9 +21,9 @@
 MANEXT =	
 
 ## fix up prefix to be a make variable
-prefix = @prefix@
-MAN =		@mandir@
-SHELL =		@SHELL@
+prefix		= @prefix@
+mandir		= @mandir@
+SHELL		= @SHELL@
 
 
 all:
@@ -52,25 +52,25 @@
 # Standard man install target.
 #
 install:
-	-${SRC}/mkinstalldirs $(MAN)/man1 $(MAN)/man5 $(MAN)/man8
-	for i in ${SRC}/*.1; do $(MANINSTCMD) $$i $(MAN)/man1/`basename $$i`$(MANEXT); done
-	for i in ${SRC}/*.5; do $(MANINSTCMD) $$i $(MAN)/man5/`basename $$i`$(MANEXT); done
-	for i in ${SRC}/*.8; do $(MANINSTCMD) $$i $(MAN)/man8/`basename $$i`$(MANEXT); done
+	-${SRC}/mkinstalldirs $(DESTDIR)$(mandir)/man1 $(DESTDIR)$(mandir)/man5 $(DESTDIR)$(mandir)/man8
+	for i in ${SRC}/*.1; do $(MANINSTCMD) $$i $(DESTDIR)$(mandir)/man1/`basename $$i`$(MANEXT); done
+	for i in ${SRC}/*.5; do $(MANINSTCMD) $$i $(DESTDIR)$(mandir)/man5/`basename $$i`$(MANEXT); done
+	for i in ${SRC}/*.8; do $(MANINSTCMD) $$i $(DESTDIR)$(mandir)/man8/`basename $$i`$(MANEXT); done
 
 # Install man pages in compressed form.
 #
 install.zman:
 	for i in ${SRC}/*.1; do compress < $$i >/tmp/_; \
-		 $(MANINSTCMD) /tmp/_ $(MAN)/man1.Z/$$i$(MANEXT); rm -f /tmp/_; done
+		 $(MANINSTCMD) /tmp/_ $(mandir)/man1.Z/$$i$(MANEXT); rm -f /tmp/_; done
 	for i in ${SRC}/*.5; do compress < $$i >/tmp/_; \
-		 $(MANINSTCMD) /tmp/_ $(MAN)/man5.Z/$$i$(MANEXT); rm -f /tmp/_; done
+		 $(MANINSTCMD) /tmp/_ $(mandir)/man5.Z/$$i$(MANEXT); rm -f /tmp/_; done
 	for i in ${SRC}/*.8; do compress < $$i >/tmp/_; \
-		 $(MANINSTCMD) /tmp/_ $(MAN)/man8.Z/$$i$(MANEXT); rm -f /tmp/_; done
+		 $(MANINSTCMD) /tmp/_ $(mandir)/man8.Z/$$i$(MANEXT); rm -f /tmp/_; done
 
 uninstall:
-	for i in *.1; do rm -f $(MAN)/man1/$$i$(MANEXT); done
-	for i in *.5; do rm -f $(MAN)/man5/$$i$(MANEXT); done
-	for i in *.8; do rm -f $(MAN)/man8/$$i$(MANEXT); done
+	for i in *.1; do rm -f $(mandir)/man1/$$i$(MANEXT); done
+	for i in *.5; do rm -f $(mandir)/man5/$$i$(MANEXT); done
+	for i in *.8; do rm -f $(mandir)/man8/$$i$(MANEXT); done
 	
 realclean mostlyclean distclean:: clean
 	rm -f Makefile
diff -Nru LPRng-3.6.12/src/Makefile.in LPRng-3.6.12.new/src/Makefile.in
--- LPRng-3.6.12/src/Makefile.in	Thu Oct 28 03:30:11 1999
+++ LPRng-3.6.12.new/src/Makefile.in	Thu Nov 25 07:44:08 1999
@@ -57,17 +57,17 @@
 
 ### PATHS ####
 # prefix
-prefix=@prefix@
+prefix			= @prefix@
 # exec prefix
-exec_prefix=@exec_prefix@
+exec_prefix		= @exec_prefix@
 # for binaries
-bindir=@bindir@
+bindir			= @bindir@
 # for admin type of binaries
-sbindir=@sbindir@
+sbindir			= @sbindir@
 # for the filters
-libexecdir=@libexecdir@
+libexecdir		= @libexecdir@
 # for the configuration stuff
-sysconfdir=@sysconfdir@
+sysconfdir		= @sysconfdir@
 # locale
 localedir = $(prefix)/@DATADIRNAME@/locale
 
@@ -399,29 +399,29 @@
 
 install: all
 	echo "SETUID_ROOT IS ${PERMS}, PERMS $(ROOT_PERMS)";
-	for i in ${bindir} ${sbindir} ${LPD_DIR} ${FILTER_DIR}; do \
+	for i in $(DESTDIR)$(bindir) $(DESTDIR)$(sbindir) $(DESTDIR)$(LPD_DIR) $(DESTDIR)$(FILTER_DIR); do \
 	  if [ ! -d $$i ] ; then ${SRC}/mkinstalldirs $$i ; fi; \
 	done;
 
-	${INSTALLCMD} ${STRIP} -m $(ROOT_PERMS) lpq ${bindir}
-	${INSTALLCMD} ${STRIP} -m $(ROOT_PERMS) lprm ${bindir}
-	${INSTALLCMD} ${STRIP} -m $(ROOT_PERMS) lpr ${bindir}
-	${INSTALLCMD} ${STRIP} -m $(ROOT_PERMS) lpstat ${bindir}
-	${INSTALLCMD} ${STRIP} -m $(ROOT_PERMS) lpc ${sbindir}
-	${INSTALLCMD} ${STRIP} -m $(ROOT_PERMS) lpd ${LPD_DIR}
-	${INSTALLCMD} ${STRIP} -m $(NORM_PERMS) lpf ${FILTER_DIR}
-	${INSTALLCMD} ${STRIP} -m $(NORM_PERMS) lpbanner ${FILTER_DIR}
-	${INSTALLCMD}    -m $(NORM_PERMS) pclbanner ${FILTER_DIR}
-	${INSTALLCMD}    -m $(NORM_PERMS) psbanner ${FILTER_DIR}
-	${INSTALLCMD} ${STRIP} -m $(NORM_PERMS) checkpc ${sbindir}
-	${INSTALLCMD} ${STRIP} -m $(NORM_PERMS) lpraccnt ${sbindir}
-	cd ${bindir}; rm -f lp cancel; ln ${STRIP} lpr lp; ln -s lprm cancel;
+	${INSTALLCMD} ${STRIP} -m $(ROOT_PERMS) lpq $(DESTDIR)$(bindir)
+	${INSTALLCMD} ${STRIP} -m $(ROOT_PERMS) lprm $(DESTDIR)$(bindir)
+	${INSTALLCMD} ${STRIP} -m $(ROOT_PERMS) lpr $(DESTDIR)$(bindir)
+	${INSTALLCMD} ${STRIP} -m $(ROOT_PERMS) lpstat $(DESTDIR)$(bindir)
+	${INSTALLCMD} ${STRIP} -m $(ROOT_PERMS) lpc $(DESTDIR)$(sbindir)
+	${INSTALLCMD} ${STRIP} -m $(ROOT_PERMS) lpd $(DESTDIR)$(LPD_DIR)
+	${INSTALLCMD} ${STRIP} -m $(NORM_PERMS) lpf $(DESTDIR)$(FILTER_DIR)
+	${INSTALLCMD} ${STRIP} -m $(NORM_PERMS) lpbanner $(DESTDIR)$(FILTER_DIR)
+	${INSTALLCMD}    -m $(NORM_PERMS) pclbanner $(DESTDIR)$(FILTER_DIR)
+	${INSTALLCMD}    -m $(NORM_PERMS) psbanner $(DESTDIR)$(FILTER_DIR)
+	${INSTALLCMD} ${STRIP} -m $(NORM_PERMS) checkpc $(DESTDIR)$(sbindir)
+	${INSTALLCMD} ${STRIP} -m $(NORM_PERMS) lpraccnt $(DESTDIR)$(sbindir)
+	cd $(DESTDIR)$(bindir); rm -f lp cancel; ln ${STRIP} lpr lp; ln -s lprm cancel;
 
 uninstall:
-	rm -f ${LPD_DIR}/lpd
-	cd ${FILTER_DIR}; rm -f lpf lpbanner pclbanner psbanner
-	cd ${bindir}; rm -f lp lpq lpstat lpr lp cancel;
-	cd ${sbindir}; rm -f lpc checkpc lpraccnt;
+	rm -f $(DESTDIR)$(LPD_DIR)/lpd
+	cd $(DESTDIR)$(FILTER_DIR); rm -f lpf lpbanner pclbanner psbanner
+	cd $(DESTDIR)$(bindir); rm -f lp lpq lpstat lpr lp cancel;
+	cd $(DESTDIR)$(sbindir); rm -f lpc checkpc lpraccnt;
 
 
 ###########################################################################
