--- lprng-3.9.0/po/Makefile.am.orig	2023-03-17 22:26:01.000000000 +0100
+++ lprng-3.9.0/po/Makefile.am	2024-09-20 19:01:18.979712048 +0200
@@ -17,6 +17,8 @@ LINGUAS = de fr pl vi
 POTFILES.in = \
 	src/common/accounting.c\
 	src/common/controlword.c\
+	src/common/debug.c
+	src/common/getopt.c
 	src/common/linelist.c\
 	src/common/lpc.c\
 	src/common/lpd.c\
--- lprng-3.9.0/src/common/debug.c.orig	2023-03-17 22:26:01.000000000 +0100
+++ lprng-3.9.0/src/common/debug.c	2024-09-20 19:00:11.820075883 +0200
@@ -85,22 +85,22 @@
 */
 
 	static const char *guide[] = {
-	" use on command line, or in printcap :db=... entry", 
-	" for server:",
-	"   print:     show queue (printing) actions, larger number, more information",
-	"     NUMBER     same as print+NUMBER",
-	"   lpr:       show servicing lpr actions",
-	"   lpq:       show servicing lpq actions",
-	"   lprm:      show servicing lprm actions",
-	"   network:   show low level network actions",
-	"   database:  show low level database actions",
-	"   log:       Testing.  Don't use this unless you read the code.",
-	"   test:      Testing.  don't use this unless you read the code.",
-	" for clients (lpr, lpq, etc):",
-	"   print:     show client actions, larger number, more information",
-	"     NUMBER     same as print+NUMBER",
-	"   network:   show low level network actions.",
-	"   database:  show low level database actions.",
+	N_(" use on command line, or in printcap :db=... entry"), 
+	N_(" for server:"),
+	N_("   print:     show queue (printing) actions, larger number, more information"),
+	N_("     NUMBER     same as print+NUMBER"),
+	N_("   lpr:       show servicing lpr actions"),
+	N_("   lpq:       show servicing lpq actions"),
+	N_("   lprm:      show servicing lprm actions"),
+	N_("   network:   show low level network actions"),
+	N_("   database:  show low level database actions"),
+	N_("   log:       Testing.  Don't use this unless you read the code."),
+	N_("   test:      Testing.  don't use this unless you read the code."),
+	N_(" for clients (lpr, lpq, etc):"),
+	N_("   print:     show client actions, larger number, more information"),
+	N_("     NUMBER     same as print+NUMBER"),
+	N_("   network:   show low level network actions."),
+	N_("   database:  show low level database actions."),
 		0
 	};
 void Parse_debug (const char *dbgstr, int interactive )
@@ -154,8 +154,8 @@ void Parse_debug (const char *dbgstr, in
 		if(!found && interactive ){
 		    int i;
 		    int lastflag = 0;
-		    FPRINTF (STDERR, "debug flag format: num | flag[+num] | flag=str\n");
-		    FPRINTF (STDERR, "  flag names:");
+		    FPRINTF (STDERR, _("debug flag format: num | flag[+num] | flag=str\n"));
+		    FPRINTF (STDERR, _("  flag names:"));
 		    for (i = 0; list[i].keyword; i++) {
 				if( safestrchr( list[i].keyword, '+' ) ) continue;
 				if( lastflag ){
@@ -166,10 +166,10 @@ void Parse_debug (const char *dbgstr, in
 				}
 				switch( list[i].type ){
 				case INTEGER_K:
-					FPRINTF (STDERR, "%s=num", list[i].keyword);
+					FPRINTF (STDERR, _("%s=num"), list[i].keyword);
 					break;
 				case STRING_K:
-					FPRINTF (STDERR, "%s=str", list[i].keyword);
+					FPRINTF (STDERR, _("%s=str"), list[i].keyword);
 					break;
 				case FLAG_K:
 					FPRINTF (STDERR, "%s[+N]", list[i].keyword );
@@ -181,7 +181,7 @@ void Parse_debug (const char *dbgstr, in
 			}
 		    FPRINTF (STDERR, "\n");
 		    for(i = 0; guide[i]; ++i ){
-				FPRINTF (STDERR, "%s\n", guide[i]);
+				FPRINTF (STDERR, "%s\n", _(guide[i]));
 		    }
 
 			Errorcode = JABORT;
--- lprng-3.9.0/src/common/lpd_status.c.orig	2024-09-20 18:57:43.264214013 +0200
+++ lprng-3.9.0/src/common/lpd_status.c	2024-09-20 19:03:12.309098090 +0200
@@ -466,7 +466,7 @@ void Get_queue_status( struct line_list
 	end_of_name = 0;
 	if( displayformat != REQ_DSHORT ){
 		plp_snprintf( header, sizeof(header), "%s: ",
-			Server_queue_name_DYN?"Server Printer":"Printer" );
+			Server_queue_name_DYN?_("Server Printer"):_("Printer") );
 	}
 	len = strlen(header);
 	plp_snprintf( header+len, sizeof(header)-len, "%s@%s",
--- lprng-3.9.0/src/common/lpstat.c.orig	2024-09-20 18:57:43.264214013 +0200
+++ lprng-3.9.0/src/common/lpstat.c	2024-09-20 19:20:38.723429177 +0200
@@ -69,7 +69,7 @@ int Rawformat;
 static void usage(void)
 {
 	FPRINTF( STDERR,
-"usage: %s [-A] [-d] [-l] [-r] [-R] [-s] [-t] [-a [list]]\n"
+_("usage: %s [-A] [-d] [-l] [-r] [-R] [-s] [-t] [-a [list]]\n"
 "  [-c [list]] [-f [list]] [-o [list]]\n"
 "  [-p [list]] [-P] [-S [list]] [list]\n"
 "  [-u [login-ID-list]] [-v [list]] [-V] [-n] [-Tdbgflags]\n"
@@ -92,11 +92,11 @@ static void usage(void)
 " -v [list] printer mapping *\n"
 " -V        verbose mode \n"
 " -Tdbgflags debug flags\n"
-"    * - long status format produced\n", Name);
+"    * - long status format produced\n"), Name);
 
 	{
 	char buffer[128];
-	FPRINTF( STDERR, "Security Supported: %s\n", ShowSecuritySupported(buffer,sizeof(buffer)) );
+	FPRINTF( STDERR, _("Security Supported: %s\n"), ShowSecuritySupported(buffer,sizeof(buffer)) );
 	}
 	Parse_debug("=",-1);
 	FPRINTF( STDOUT, "%s\n", Version );
