--- lprng-3.9.0/configure.ac.orig	2024-09-20 18:51:28.982908335 +0200
+++ lprng-3.9.0/configure.ac	2024-09-20 18:51:33.426217597 +0200
@@ -5,35 +5,29 @@ AM_INIT_AUTOMAKE([foreign subdir-objects
 
 AM_MAINTAINER_MODE
 
-AC_PREFIX_DEFAULT(/usr/local)
-
 mysaved_CFLAGSset="${CFLAGS+set}"
 AC_PROG_CC
 if test "$ac_cv_prog_gcc" = yes; then
 	if test "$mysaved_CFLAGSset" != set ; then
-		CFLAGS="-g -W -Wall -Wno-unused -Wstrict-prototypes -Wmissing-prototypes"
+		CFLAGS="-W -Wall -Wno-unused -Wstrict-prototypes -Wmissing-prototypes"
 	fi
 fi;
 AC_SYS_LARGEFILE
 
 AC_PROG_AWK
-AC_SUBST(AWK)
-AC_PATH_PROG(SED,sed)dnl
-AC_SUBST(SED)dnl
-AC_PATH_PROG(PERL,perl)dnl
-AC_SUBST(PERL)dnl
+AC_PATH_PROG(SED,sed)
+AC_PATH_PROG(PERL,perl)
 dnl -- TODO:
 dnl this is used in checkpc, why not just use "chown" there?
 dnl same with chgrp
 dnl --
-AC_PATH_PROG(CHOWN,chown)dnl
+AC_PATH_PROG(CHOWN,chown)
 AC_DEFINE_UNQUOTED(CHOWN, "$CHOWN",[full path to chown])
-AC_PATH_PROG(CHGRP,chgrp)dnl
+AC_PATH_PROG(CHGRP,chgrp)
 AC_DEFINE_UNQUOTED(CHGRP, "$CHGRP",[full path to chgrp])
-AC_PATH_PROG(PRUTIL,pr)dnl
+AC_PATH_PROG(PRUTIL,pr)
 AC_DEFINE_UNQUOTED(PRUTIL, "$PRUTIL",[full path to prutil])
-AC_PATH_PROG(OPENSSL,openssl)dnl
-AC_SUBST(OPENSSL)dnl
+AC_PATH_PROG(OPENSSL,openssl)
 AC_DEFINE_UNQUOTED(OPENSSL, "$OPENSSL",[full path to openssl])
 
 dnl dnl check to see if setuid is suppressed
