--- a/configure.in.orig	2018-11-14 20:57:29.911047886 +0100
+++ b/configure.in	2018-11-14 21:11:48.224371420 +0100
@@ -1589,7 +1589,7 @@
 		save_LDFLAGS=$LDFLAGS
 		LIBS="-lssl -lcrypto $LIBS"
 		LDFLAGS="$LDFLAGS -L$dir "
-		AC_TRY_LINK_FUNC(SSL_load_error_strings,ac_linked_libssl="true",
+		AC_TRY_LINK_FUNC(OPENSSL_init_ssl,ac_linked_libssl="true",
 			ac_linked_libssl="false");
 		AC_TRY_LINK_FUNC(RC4_set_key,ac_linked_libcrypto="true",
 			ac_linked_libcrypto="false");
