diff -Nur LPRng-3.8.28.orig/src/Makefile.in LPRng-3.8.28.as-needed/src/Makefile.in
--- LPRng-3.8.28.orig/src/Makefile.in	2003-11-17 20:35:23.000000000 +0000
+++ LPRng-3.8.28.as-needed/src/Makefile.in	2007-05-19 16:47:40.000000000 +0000
@@ -265,7 +265,7 @@
 
 
 liblpr.la: $(LIBLPR_OBJS)
-	$(LINK) $(LIBLPR_OBJS) -rpath ${libdir}
+	$(LINK) $(LIBLPR_OBJS) -rpath ${libdir} $(LIBS)
 
 lpd: $(LPD_OBJS) liblpr.la
 	$(LINK) ${LPD_OBJS} liblpr.la $(LIBS)
