--- LPRng-3.8.23/src/include/lp.h.orig	2003-09-05 22:07:21.000000000 +0200
+++ LPRng-3.8.23/src/include/lp.h	2003-11-13 08:03:56.471808920 +0100
@@ -267,7 +267,7 @@
 EXTERN char* Chooser_DYN;	/* choose the destination for a load balance queue */
 EXTERN int Chooser_interval_DYN;	/* interval between tests for load balance destination */
 EXTERN int Chooser_scan_queue_DYN;	/* scan the queue */
-EXTERN char* Chooser_routine_DYN;	/* choose the destination for a load balance queue */
+EXTERN int Chooser_routine_DYN;	/* choose the destination for a load balance queue */
 EXTERN int Class_in_status_DYN;	/* Show class in status information */
 EXTERN char* Comment_tag_DYN; /* comment identifying printer (LPQ) */
 EXTERN char* Config_file_DYN;
@@ -392,7 +392,7 @@
 EXTERN char* Prefix_Z_DYN; /* prefix -Z options on outgoing or filter*/
 EXTERN char* Prefix_option_to_option_DYN; /* prefix option to option, ie, "z,o" */
 EXTERN char* Printcap_path_DYN;
-EXTERN char* Printer_DYN;		/* Printe r name for logging */
+/* EXTERN char* Printer_DYN; duplicate??? */		/* Printe r name for logging */
 EXTERN char* Printer_DYN;	/* printer name */
 EXTERN char* Printer_perms_path_DYN;
 EXTERN char* Queue_name_DYN;	/* Queue name used for spooling */
