--- LPRng-3.8.15/Makefile.in~	Sun Nov 24 00:49:04 2002
+++ LPRng-3.8.15/Makefile.in	Sun Nov 24 00:49:04 2002
@@ -117,7 +117,7 @@
 ###############################################################################
 $(ALLDIRS):
 	if [ "$@" != po -o "$(USE_NLS)" != "no" ] ; then \
-		cd $@; $(MAKE) localedir=${DESTDIR}${LOCALEDIR} \
+		cd $@; $(MAKE) localedir=${LOCALEDIR} \
 			DESTDIR=$(DESTDIR) $(MAKETARGET) ; \
 	fi
 
